### DISCUSSION TITLE
Adjustable Layer Positions in Tilemap Node

### DISCUSSION THREAD

**Discussion by TeacanTzu** [godotengine/godot-proposals#8659] [2D]
**Proposal:**
Feature Overview:
Introducing a feature that allows to change the position or offset of individual layers within the TileMap node. This would be particularly valuable for those working on 2D isometric games, enabling the stacking of vertical tiles to create a sense of height while only using one TileMap node.

I've added an attachment of how the UI might look. The word "Position" could be replaced with "Offset"

![TileMapLayerOffset](https://github.com/godotengine/godot-proposals/assets/132375349/7e4d5cb1-9da6-44a7-aef7-7666a39169c4)

This would help to create maps like this which is currently not possible without using multiple nodes.

![Example](https://github.com/godotengine/godot-proposals/assets/132375349/d7479842-2fbd-495d-a442-ff4d70533e8f)

**Comment by AThousandShips** [comment]
Duplicate of:
* https://github.com/godotengine/godot-proposals/discussions/8196

Please search for open discussions before opening a new one 🙂

**Reply by TeacanTzu** [reply]
I thought the discussion was 4 years old but it was just the american way to write dates, oops.

**Reply by AThousandShips** [reply]
That doesn't matter, don't open new ones because the old ones are old 🙃
